.storybook/addons/codeEditorAddon/codeAddon.js

```diff
@@ -65,8 +68,7 @@ export const withCodeEditor = makeDecorator({
   name: `withCodeEditor`,
   parameterName: 'myParameter',
   skipIfNoParametersOrOptions: false,
-  wrapper: (getStory, context, { options }) => {
```

Comment:
@sebastienlevert This change is removing the theme toggle from the stories, we probably don't want to do that

Reply:
Does it? I see the disableThemeToggle options in there. I'm confused.

Reply:
Now I understand. The code was pushed below. I'll bring it up.